src/components/Account/Avatar.js

```diff
@@ -26,7 +26,18 @@ const Avatar = props => {
 
   const postUploadTask = url => {
     setImgURL(url);
+
     //link the file to the user
+    props.firebase.auth.currentUser
+      .updateProfile({
+        photoURL: url,
+      })
+      .then(function() {
```

Comment:
Maybe write some console output here; now the then and catch of this Promise do nothing.